server/JsDbg.Gdb/GdbDebugger.cs

```diff
@@ -36,6 +35,31 @@ class GdbDebugger : IDebugger {
             }
         }
 
+        void NotifyDebuggerChange(DebuggerChangeEventArgs.DebuggerStatus status) {
+            this.DebuggerChange?.Invoke(this, new DebuggerChangeEventArgs(status));
+        }
+
+        void HandleGdbEvent(object sender, string ev) {
+            if (ev[0] != '%')
+                return;
+
+            bool oldState = debuggerBusy;
+
+            if (ev == "%cont")
+                debuggerBusy = true;
+            else if (ev == "%stop" || ev == "%exit")
+                debuggerBusy = false;

```

Comment:
It doesn't seem like we'll notify on the change from `Break` to `Detaching` -- is that intentional?

Reply:
I think you're right that we should, I'll fix that tomorrow.

BTW, can you confirm that I understood the semantics correctly? I.e. when you "continue" from a breakpoint we should send a waiting event?

Reply:
Yeah...`Waiting` is weird.  As you probably saw, in the WinDbg version it gets sent when we're polling for WinDbg to become available while trying to serve a request from the client.  On the client, when it gets the waiting event it throws up a full-page backdrop to say that progress is blocked by the fact that the debugger is running.  So sending it on every "continue" might cause more flickering than would be ideal? But it's not unreasonable.

Reply:
OK, changes made. Please take another look!